package.json

```diff
@@ -49,13 +47,14 @@
     "seneca-promisify": ">=3"
   },
   "devDependencies": {
-    "node-fetch": "^2",
     "@seneca/doc": "^4.1.0",
     "@types/jest": "^29.0.1",
     "@types/node": "^18.7.16",
+    "@types/node-fetch": "^2.6.2",
     "esbuild": "^0.15.7",
     "esbuild-jest": "^0.5.0",
     "jest": "^29.0.3",
+    "node-fetch": "^2",
```

Comment:
is this a dev dep? surely it is needed in production?

Reply:
Indeed, i was using only for typings, but node already has one. Removed.

Reply:
now the dependency has moved to @seneca/provider in any case